packages/manager/apps/telecom/src/app/telecom/pack/move/address/future/move-address-future.html

```diff
@@ -0,0 +1,106 @@
+<div class="col-md-5 col-sm-12 col-xs-12 m-0 p-0">
+    <div class="block-emphasis location location--future">
```

Comment:
Where does those classes come from ?

Reply:
From the old version of the template. I remove it.